static/templates/settings/organization_settings_admin.hbs

```diff
@@ -192,8 +192,11 @@
                     </div>
                 </div>
 
-                {{> default_code_language_settings_widget
-                  label=admin_settings_label.realm_default_code_language }}
+                {{> dropdown_list_widget
+                  setting_name="realm_default_code_block_language"
+                  list_placeholder=(t 'Filter languages')
+                  reset_button_text=(t '[Unset]')
```

Comment:
I haven't seen this translation tag request approach before, and I don't think it works.

(At least, if I run `manage.py makemessages`, `Filter languages` doesn't appear in the resulting translation.json).

Reply:
That's weird, this should have worked.

At least on the handlebars side of things, we are trying to read the translations data:

![image](https://user-images.githubusercontent.com/8033238/79418738-fcd68b80-7fd2-11ea-95d6-b05140d2b879.png)